README.md

```diff
@@ -207,6 +207,54 @@ docker-compose run app yarn lint:diff
 docker-compose run app node_modules/.bin/eslint --fix path/to/file.js
 ```
 
+## Feature Flags
+Environment-specific feature flags are supported for both the api and frontend. Flagged features are assumed to be "off" unless the flag exists (and the value is truthy), thus feature flag conditions should always check for the presence or truthiness of the flag, not for it's absence. Environment variables for feature flags *MUST* be prefixed by `FEATURE_`, ex. `FEATURE_BRING_THE_AWESOME`.
+
+### Api feature flags
+Api feature flags are evaluated at *runtime* and should be created explicitly in the code before the corresponding environment variable can be used. Example:
+
+Given environment variable `FEATURE_AWESOME_SAUCE='true'`
+
+1. Add the flag to the known flags:
+```js
+// api/features.js
+const Flags = {
+  //...
+  AWESOME_SAUCE: 'AWESOME_SAUCE',
+}
+```
+2. Check if the feature is enabled:
+```js
+// some code in a child folder of /api
+const Features = require('../features');
+
+if (Features.enabled(Features.Flags.AWESOME_SAUCE)) {
+  doSomething();
+}
+```
+
+### Frontend feature flags
+Frontend feature flags are evaluated at *compile* time NOT at runtime, resulting in unused codepaths being exluded from the built code. Environment variables with the `FEATURE_` prefix are available globally within the codebase at compile time. This magic is performed by `webpack.DefinePlugin` and minification.
+
+
+Example:
+
+Given environment variable `FEATURE_AWESOME_SAUCE='true'`
+
+```js
+if (FEATURE_AWESOME_SAUCE === 'true') {
```

Comment:
For consistency, what are you thoughts on webpack's `getFeatureFlags` removing `FEATURE_` from the env variable like the api flags or vice versa?

Reply:
I def want to leave the `FEATURE_` part of the env var on the client so it's clear what the global is and so there are no potential collisions.